mmv1/products/dlp/JobTrigger.yaml

```diff
@@ -720,15 +727,9 @@ properties:
             properties:
               - !ruby/object:Api::Type::String
                 name: 'startTime'
-                at_least_one_of:
```

Comment:
could this be more accurate to be configured as 
```
                at_least_one_of:
                  - inspect_job.0.storage_config.0.timespan_config.0.start_time
                  - inspect_job.0.storage_config.0.timespan_config.0.end_time
                  - inspect_job.0.storage_config.0.timespan_config.0.enable_auto_population_of_timespan_config
```
rather than fully removing it?

Or is the enable_auto... field an optional+computed field that would automatically flag true if the others were not supplied?

Reply:
`start_time` and `enable_auto_population_of_timespan_config` are mutually exclusive (at least if we follow the console UI behaviour), but, if i don't supply `start_time` / `end_time`, `enable_auto_population_of_timespan_config¡ can be set to false (which is the default if not set), if i want to scan all the content every time.

We have 4 configuration cases, each for storage type target (using the relevant configuration block).

When i use

- `bigquery_options`=> in `timespan_config` configuration block `timestamp_field`is required
- `cloud_storage_options` => in `timespan_config` configuration block `timestamp_field`is NOT required, the other 3 are optional
- `datastore_options` => in `timespan_config` configuration block `timestamp_field` seems to be optional, but if i create a job trigger using console UI i can't specify this if field, if i choose "Datastore" as storage type, the same for the other 3 fields
- `hybrid_options` => `timespan_config` configuration block not required

In any case your suggestion make sense, because if i specify the `timespan_config¡ block i cannot leave it empty because it would not make sense (i've already tried this case locally, i can apply the resource but terraform complaints with a perpetual diff, it would set this `timespan_config {}` empty block)

I try to figure out how to satisfy all the cases, i've seen another useful field, `required_with` (in https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/api/type.rb) that could be used here.